pkg/dockerclient/mock.go

```diff
@@ -0,0 +1,69 @@
+package dockerclient
```

Comment:
Call this file `mock_test.go` so it won't be built in the regular build.

Reply:
Used in other modules + should not be linked in